src/main/java/longtimenosee/logic/parser/FindCommandParser.java

```diff
@@ -36,59 +48,144 @@ public FindCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_ADDRESS, PREFIX_EMAIL,
-                        PREFIX_TAG);
+                        PREFIX_TAG, PREFIX_BIRTHDAY, PREFIX_INCOME, PREFIX_RISK_APPETITE, PREFIX_TITLE,
+                        PREFIX_COVERAGES, PREFIX_COMPANY);
 
         if (!isAtLeastOnePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_PHONE, PREFIX_ADDRESS, PREFIX_EMAIL,
-                PREFIX_TAG)
+                PREFIX_TAG, PREFIX_BIRTHDAY, PREFIX_INCOME, PREFIX_RISK_APPETITE, PREFIX_TITLE, PREFIX_COVERAGES,
+                PREFIX_COMPANY)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
 
         List<Predicate<Person>> predicates = new ArrayList<Predicate<Person>>();
```

Comment:
Good use of polymorphism to abstract out the appropriate prefixes for person into a list!